scripts/download_technique_animations.py

```diff
@@ -45,6 +44,7 @@ def process_filename(filepath: str) -> str:
     """Extract base filename from an animation file path."""
     cleaned_filename = os.path.splitext(os.path.basename(filepath))[0]
     cleaned_filename = cleaned_filename.strip("0123456789_")
+    cleaned_filename = cleaned_filename.split("px-")[-1]
```

Comment:
Doesn't this require px to be present? You can use partition instead.

Reply:
Nope, it would work without as well:
```python
>>> a_str = "firelion_back.git"
>>> a_str.split("px-")[-1]
'firelion_back.git'
```
I will give partition a try, though :).

Reply:
Weird, wonder why I thought that.